### DISCUSSION TITLE
Chat in PDA

### DISCUSSION THREAD

**Discussion by xsainteer** [space-wizards/space-station-14#24919] [Ideaguys]
I got idea I can't make myself, therefore asking you guys to make chat system in PDA so a player can interact with another player 1v1 with nobody seeing messages but the players chatting. There's also the idea of adding to Sec the possibility to watch them chats if needed

**Comment by TK-A369** [comment]
There already is a [design document](https://docs.spacestation14.com/en/proposals/julian-vasilis-pda-messaging.html) about something like that.
